clients/unshaded/src/main/java/tachyon/client/UfsUtils.java

```diff
@@ -93,7 +93,7 @@ private static void loadUfs(TachyonURI tfsAddrRootPath, TachyonURI ufsAddrRootPa
    * @throws IOException when an event that prevents the operation from completing is encountered
    */
   @Deprecated
-  public static void loadUnderFs(TachyonFS tfs, TachyonURI tachyonPath, TachyonURI ufsAddrRootPath,
+  public static void loadUfs(TachyonFS tfs, TachyonURI tachyonPath, TachyonURI ufsAddrRootPath,
```

Comment:
I see, I think we need to change the arguments of the shell script then, since mount points are a separate concept now. We can do this in another PR though.